source/cloud/cloud-administration/common-support-packet.rst

```diff
@@ -0,0 +1,22 @@
+Use the System Console to generate a Mattermost Support Packet that includes configuration information, logs, and data on external dependencies. Confidential data, such as passwords, are automatically stripped. 
+
+.. note:: 
+
+  System Admins should always remove sensitive information before sending the support packet to Mattermost by email or via a support ticket. 
+
+To generate a support packet:
+
+1. Go to System Console, then select **Commercial Support** from the **Main Menu**.
+2. Select **Download Support Packet**. A zip file is downloaded to the local machine.
+
+A Mattermost Support Packet contains the following five files:
+
+- mattermost.log
+- plugins.json
+- sanitized_config.json
+- support_packet.yaml
+- warning.txt
+
+You are notified if any packet files are unavailable during packet generation. See the ``warning.txt`` file for details.
+
+3. Sanitize configuration and log files in the Support Packet files to remove usernames, passwords, and LDAP groups. Replace these details with example strings that contain the same special characters if possible, as special characters are common causes of configuration errors.
```

Comment:
We mention that passwords are automatically stripped in the opening paragraph, but mention here that they should be removed. I'm also wondering whether there's a specific method we recommend in terms of removing this information, or if it's something a System Admin would be inherently familiar with?

Reply:
@justinegeffen This is a great set of questions worth exploring with R&D further. If we can provide thought leadership and guidance to System Admins, they can complete these tasks faster and with greater confidence. 

@hahmadia  - Can you clarify what specifically gets sanitized when the support packet is generated, and what we recommend System Admins should do to further sanitize manually prior to sending the support packet to Mattermost? Thanks!

Reply:
Sure thing! 

Nothing in `mattermost.log` , `plugins.json`, `support_packet` or `warning.txt` get sanitized or removed.

The following values in the `config.json` get sanitized:

- LdapSettings.BindPassword
- FileSettings.PublicLinkSalt
- FileSettings.AmazonS3SecretAccessKey
- EmailSettings.SMTPPassword
- GitLabSettings.Secret
- GoogleSettings.Secret 
- Office365Settings.Secret
- OpenIdSettings.Secret 
- SqlSettings.DataSource
- SqlSettings.AtRestEncryptKey 
- ElasticsearchSettings.Password
- All SqlSettings.DataSourceReplicas
- All SqlSettings.DataSourceSearchReplicas
- MessageExportSettings.GlobalRelaySettings.SmtpPassword
- ServiceSettings.GfycatApiSecret 
- ServiceSettings.SplitKey

If they like and feel this isn't enough, they can sanitize / remove whatever other data they want.

The  LDAP groups are not generated for us to view. Only LDAP Version and LDAP Vendor are generated if they are available in the `support_packet.yaml`

As for admins doing anything further, I think that's not required. I'd just recommend them to skim through the `plugins.json` and ensure their plugins don't have any sort of confidential information that needs to be hidden (like secret keys) because plugins are not sanitized but other than that I think they should be okay.

Reply:
Thank you for the great details, @hahmadia! I've requested a second review from you to ensure that I've captured these details accurately in the product documentation.